firefox-ios/Client/Frontend/Browser/Tabs/State/TabTrayState.swift

```diff
@@ -134,14 +138,25 @@ struct TabTrayState: ScreenState, Equatable {
             // Only update the nomal tab count if the tabs being refreshed are not private
             let tabModel = context.tabDisplayModel
             let isPrivate = tabModel.tabs.first?.isPrivate ?? false
-            let tabCount = tabModel.normalTabsCount
+            let tabCount = isPrivate ? state.normalTabsCount : tabModel.normalTabsCount
```

Comment:
what's the difference between  `state.normalTabsCount` and  `tabModel.normalTabsCount`?

Reply:
It's possible tabModel.normalTabsCount might not be calculated in certain cases if it's updating the private window so just safer to keep the value the same as it's current value since the private screen cannot impact this number for the normal screen.